pkg/action/bootentries_test.go

```diff
@@ -682,7 +682,7 @@ var _ = Describe("Bootentries tests", Label("bootentry"), func() {
 			It("fails to select the boot entry if it doesnt exist", func() {
 				err := SelectBootEntry(config, "kairos")
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("does not exist"))
+				Expect(err.Error()).To(ContainSubstring("failed to get any required GRUB configuration file"))
```

Comment:
I see, if this is erroring out and blocking the feature, its becuase there is not enough mocking around. we relied in the reader returning an empty list to check against this but this wont work anymore.